lib/Fhp/Segment/CAZ/GebuchteCamtUmsaetze.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Fhp\Segment\CAZ;
+
+use Fhp\Segment\BaseDeg;
+
+/**
+ * Needed for
+ * Issue: GetStatementOfAccountXML receiving multiple CAMT XML files
+ * @link: https://github.com/nemiah/phpFinTS/issues/370
+ *
+ * @link: https://github.com/nemiah/phpFinTS/pull/371
+ */
+class GebuchteCamtUmsaetze extends BaseDeg
+{
+    /** Allow for up to 999 Binary fields (XML files) */
+    /** @var Bin[] @Max(999) */
+    public $gebuchteCamtUmsaetze;
+
+    /**
+     * Array of strings to store XML srings from the Bins structure.
+     * @var string[]
+     */
+    private $xml;
+
+    /**
+     * Gets the binary data as array of strings.
```

Comment:
If you express this as `@returns string[]`, then many IDEs know how to deal with it. (And the English explanation is then redundant.)